server/services/timelineService.ts

```diff
@@ -1,14 +1,9 @@
 import type { Timeline, TimelineEvent } from 'viewModels'
-import type { TimelineEventResponse } from 'educationAndWorkPlanApiClient'
 import EducationAndWorkPlanClient from '../data/educationAndWorkPlanClient'
-import { toTimeline } from '../data/mappers/timelineMapper'
+import toTimeline from '../data/mappers/timelineMapper'
```

Comment:
This is where we used to do the filtering of event types. It's now done in the mapper above ☝️